skyvern/cli/run_commands.py

```
@@ -163,12 +163,103 @@ def run_ui() -> None:
     asyncio.run(start_services())
 
 
+@run_app.command(name="dev")
+def run_dev() -> None:
+    """Run the Skyvern API server and UI server in the background (detached).
+
+    This command starts both services and immediately returns control to your terminal.
+    Use 'skyvern stop all' to stop the services.
+    """
+    load_dotenv(resolve_backend_env_path())
+    from skyvern.config import settings as skyvern_settings  # noqa: PLC0415
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)